baseframe/templates/baseframe/bootstrap3/autoform.html.jinja2

```diff
@@ -12,7 +12,7 @@
   {{ widget_ext_scripts(form) }}
   <script type="text/javascript">
     $(function() {
-      {{ widgetscripts(form, script=false) }}
+      {{ widgetscripts(form, script=false, form_ref_id=ref_id) }}
```

Comment:
Use `ref_id=ref_id`. Don't change the parameter name unless there is good reason to. If `ref_id` is ambiguous, it should be renamed to `form_ref_id` everywhere.